src/components/NavBar/components/SubDrop/index.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
remove eslint rule and use a div so it can have a key

Reply:
done